### DISCUSSION TITLE
TTS bug in Dify

### DISCUSSION THREAD

**Discussion by sawyer-shi** [langgenius/dify#10739] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

There is a major bug in Dify. If you have deployed the TTS A model and then want to switch to another TTS B model, it keeps looking for the old model TTS A, which I have either disabled or deleted, and cannot automatically switch to TTS B. There is also no place to specify this.
![1](https://github.com/user-attachments/assets/578005e5-5102-4d8d-b064-5c102eb76dad)
![2](https://github.com/user-attachments/assets/4216a621-5855-477c-a00a-f8842de1c5df)

**Comment by MrWangChong** [comment]
How did you solve this problem? I have also encountered this problem

**Reply by sawyer-shi** [reply]
just change your default system TTS model to the new one in page system setting

**Reply by MrWangChong** [reply]
> just change your default system TTS model to the new one in page system setting

Thank you for your answer. I have also set it up, but I have changed a few models. When converting text to speech, it always rotates in circles. Currently, I am using ChatTTS

![image](https://github.com/user-attachments/assets/031b1d61-ff40-4577-8754-095576b669a5)
